src/reduced_basis/rb_construction_base.C

```diff
@@ -673,18 +663,18 @@ void RBConstructionBase<Base>::generate_training_parameters_deterministic(const
             for (indices[2]=0; indices[2]<n_training_samples_per_param[2]; indices[2]++)
               {
                 unsigned int param_count = 0;
-                for (const std::string & param_name : param_names)
+                for (const auto & [param_name, _] : min_parameters)
```

Comment:
We don't currently use underscore elsewhere in the library to refer to unused parameters. Although I think it might become standardized at some point, I'd prefer to use `pr` and `pr.first` for consistency for now. For now, it doesn't even save much (if any) code since we still have to explicitly libmesh_ignore it.